### PR TITLE
EOS-25061: Execute utils_setup as a binary

### PR DESCRIPTION
**Pull request by selvakumaar5496**
Signed-off-by: Selvakumar <selva.k.nambi@seagate.com>

# Problem Statement
- https://jts.seagate.com/browse/EOS-25061

# Design
-  Remove argv from function call 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [x] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -203,11 +203,11 @@ def post_install(post_install_template: str):
         Conf.load(post_install_template_index, post_install_template)
 
         machine_id = Conf.machine_id
-        key_list = [f'server_node>{machine_id}>hostname', f'server_node>{machine_id}>name']
+        key_list = [f'node>{machine_id}>hostname', f'node>{machine_id}>name']
```
Comment (selvakumaar5496) [inline py-utils/src/setup/utils.py:206]: Why was this not taken care on changing ConfStore keys @vastradparayya

Comment (sachinpunadikar) [reply py-utils/src/setup/utils.py:206]: It can be due to merging main into kubernetes as well

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by vastradparayya — APPROVED** [review]
LGTM

**Pull request merged**